### ISSUE TITLE
Cryptic error message from curve_interval

### ISSUE THREAD

**Issue by joshua-goldberg** [mjskay/ggdist#173]
Hi there!

I've been messing around with `curve_interval`s to summarize data, but am encountering a rather unhelpful error message:  `Error in rowMeans(pointwise_depths, na.rm = na.rm) : 'x' must be an array of at least two dimensions`. I'm willing to believe that my data may not be a good fit for a curve-wise interval (or at least that `point_interval` is the appropriate option), but I'd appreciate some combination of a more informative error message and/or more examples of `curve_interval`(perhaps including the .along argument?) that illustrate use cases. 

My data are not evenly or identically distributed along the 'x' variable across curves, which seems to be problematic. Happy to come up with a reprex if that's helpful to diagnose the issue, but this may be known/intended/desirable behavior, so I'll save a bit of digital ink for the moment.

Cheers,
J

**Comment by mjskay** [comment]
Yeah that's a bit odd --- a reprex would help me diagnose. Thanks!
